packages/cli/src/util/pruneOldFilesInDir.ts

```diff
@@ -1,17 +1,25 @@
 import fs from "node:fs";
 import path from "node:path";
 
-export function pruneOldFilesInDir(dirpath: string, maxAgeMs: number): void {
+export function pruneOldFilesInDir(dirpath: string, maxAgeMs: number): number {
+  let deletedFiles = 0;
```

Comment:
Maybe name it `numDeleteFiles`? Same to the caller in `handler.ts`

Reply:
not a fan of hungarian notation, why do we need this?

Reply:
> not a fan of hungarian notation, do we really need this here?

My first thought on this variable is it stores an array of file names. But if this is a convention we adopt I am cool with it.

Reply:
> But if this is a convention we adopt I am cool with it.

We should use descriptive variable names, if the name is not descriptive enough we should fix that. Using hungarian notation is something we should avoid imo, having the type of a variable as part of the name should not be required, especially when using a typed language.

In general, we have quite verbose function and variable names in a lot of places which I personally like.

What about `totalDeletedFiles`? <-- this even adds slightly more information to the variable, or `deletedFileCount`?

Reply:
`deletedFileCount` sounds good 👍